javascript/net/grpc/web/generator/grpc_generator.cc

```diff
@@ -852,10 +852,12 @@ void PrintProtoDtsMessage(Printer* printer, const Descriptor* desc,
 
   for (int i = 0; i < desc->oneof_decl_count(); i++) {
     const OneofDescriptor* oneof = desc->oneof_decl(i);
-    vars["js_oneof_name"] = ToUpperCamel(ParseLowerUnderscore(oneof->name()));
-    printer->Print(
-        vars, "get$js_oneof_name$Case(): $class_name$.$js_oneof_name$Case;\n");
-    printer->Print("\n");
+    if (!oneof->is_synthetic()) {
```

Comment:
Btw nice job finding the `is_synthetic()` API!😃

Reply:
Thanks, you forget what it's like to find docs via Google instead of using your IDE 😂

Reply:
Yes right.. 😄